src/dex-ui/pages/Pools/Pools.tsx

```diff
@@ -65,8 +65,7 @@ const Pools = (): JSX.Element => {
   const [colHeadersState, setState] = useState({ allPoolsColHeaders, userPoolsColHeaders });
 
   const navigate = useNavigate();
-  const [searchParams] = useSearchParams();
-  const [queryParamState, setQueryParamState] = useState({
+  const [poolsParamsState, setPoolsParamsState] = useState({
```

Comment:
What's the purpose of storing these state variable in the UI layer? The pools slice should provide updates to these values since you're managing them in the zustand slice now (unless I'm missing something).